README.md

```diff
@@ -1,2 +1,28 @@
-# news-for-maintainers
-Announcements of changes impacting collection contributors and maintainers
+# News for Maintainers
+Announcements of changes impacting Ansible collection contributors and maintainers.
+
+[Current open issues](https://github.com/ansible-collections/news-for-maintainers/issues).
+
+This repository is created in accordance with the [Community decision](https://github.com/ansible-community/community-topics/issues/51) as a replacement for [Issue 45](https://github.com/ansible-collections/overview/issues/45) to track changes that Ansible collection maintainers should be aware of.
+
+If you are a collection maintainer, please subscribe to this repository using the `Watch` button in the upper right corner of the GitHub web interface.
+
+The news are made via issues - one per announcement.
+
+# Examples of news posted via this repository
+
+- Updates made to ``ansible-test`` to detect new issues.
+- New OS distros available for testing.
+- Changes in test containers.
+- Calls for CI test matrix updates to prevent failing tests.
+- Changes/reminders about freeze and version bump dates.
+- News and required actions related to libraries' deprecation.
+- Changes to `ansibullbot` workflow.
+- Pull request review days and contributor summit announcements.
+- Changes in the changelog format.
+- Collection documentation markup changes.
+
+# Guidelines
+
+- Please do **not** create issues to ask questions. Join us on [Matrix or IRC](https://docs.ansible.com/ansible/latest/community/communication.html#real-time-chat) instead.
+- Please avoid unnecessary discussions in issues. Every comment posted will notify **a lot** of folks! You can add reactions to an issue/a comment instead for feedback that does not annoy everyone.
```

Comment:
While I agree with this, it might be better to provide an alternative for those who _do_ want to discuss it. I wonder if this is an opportunity to use the Discussions feature in the repo, where the Issue is used for announcement, and the Discussion is where discussion/conversation takes place.

This would work well if the notifications we can subscribe to are granular enough to get notified of new issues but not new discussions.

This doesn't have to be decided right now, nor is it a blocker for this README change.

Reply:
to add, it does seem like the notifications could work nicely with this kind of split

![image](https://user-images.githubusercontent.com/1260690/150691332-b82d4b5a-f900-461a-8823-6d02e4a3ef0b.png)

Reply:
That's indeed a good suggestion and discovery! It's probably best to suggest people to only subscribe to issues (and not to discussions) if they don't want to read everything.

Reply:
Sounds like a great idea! Added. PTAL (if anyone approves it, it would be great and i could merge it and go further)